### ISSUE TITLE
MaterialInventoryLayout not found when running with Eclipse

### ISSUE THREAD

**Issue by abledbody** [jsettlers/settlers-remake#246]
I'm completely new to Java and Eclipse, so I'm wondering why it is that when I run the program it stalls on 'Waiting for Other Players'. I ran Eclipse's debug doohickey and it claims the error is in MaterialInventoryLayout, or lack thereof.

I tried turning the piece of code that called for it into a comment and then it complained that  jsettlers.graphics.ui.layout.WarriorsLayout didn't exist, as well, so I'm thinking that I'm missing some fundamental part of the code, or I skipped some instruction under the Developer's Guide.

Also, is there some place more suited to posting this kind of thing?

**Comment by codingberlin** [comment]
This sounds like that you need to run the ant task https://github.com/jsettlers/settlers-remake/blob/master/build.ant#L4 to generate the layout classes from xml.

Maybe this section out of the https://github.com/jsettlers/settlers-remake/blob/master/README.md may help:
```
Build all the projects. This is easiest done with "CTRL + B". Alternatively, you can right click the jsettlers.main.swing -> build -> build.ant and click "Run as" and chose "Ant Build". If you have trouble building the projects, the following steps might help:

    Clean and build all the projects.
    Make sure the project jsettlers.common has a source folder called "gen". If not, refresh the project. If it still isn't there, try to explicitly build the project (right click on the project -> "Build Project").
    After that, rebuild all projects (without a clean!). You can do this with "CTRL + B".
    If there is still a project refusing to build, try to explicitly build that one via right click on the project -> "Build Project".
    If you know how to fix this trouble, I'm looking forward to your hints and code changes ;).
```

**Comment by abledbody** [comment]
@codingberlin When I tried to do that I got the error

    BUILD FAILED
    C:\Users\******\******\settlers-remake-master\jsettlers.main.swing\build\build.ant:98: The following error    occurred while executing this line:
    C:\Users\******\******\settlers-remake-master\jsettlers.common\build\build.ant:26: The following error occurred while executing this line:
    C:\Users\******\******\settlers-remake-master\jsettlers.common\build\createTextures.ant:31: Compile failed; see the compiler error output for details.

**Comment by andreas-eberle** [comment]
Hi @abledbody,

can you try running the `\jsettlers.graphics\build\buildLayouts.ant` ant script? It should build these layouts (although this should already run automatically).

**Comment by abledbody** [comment]
@andreas-eberle
```
Buildfile: C:\Users\******\******\settlers-remake-master\jsettlers.graphics\build\buildLayouts.ant
loadLib:
[mkdir] Created dir: C:\Users\******\******\settlers-remake-master\jsettlers.graphics\build\build\.buildScripts
[javac] C:\Users\******\******\settlers-remake-master\jsettlers.graphics\build\buildLayouts.ant:29: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds

BUILD FAILED
C:\Users\******\******\settlers-remake-master\jsettlers.graphics\build\buildLayouts.ant:29: srcdir "C:\Users\******\******\settlers-remake-master\jsettlers.graphics\jsettlers.common\src" does not exist!

Total time: 223 milliseconds
```

**Comment by abledbody** [comment]
Issue solved because of #251.
